recipes/systemc-cci/1.0.0/conanfile.py

```diff
@@ -0,0 +1,70 @@
+from conans import ConanFile, tools, AutoToolsBuildEnvironment
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+class SystemccciConan(ConanFile):
+    name = "systemc-cci"
+    version = "1.0.0"
+    description = """SystemC Configuration, Control and Inspection library"""
+    homepage = "https://www.accellera.org/"
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "Apache-2.0"
+    topics = ("simulation", "modeling", "esl", "cci")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False]
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True
+    }
+    requires = "systemc/2.3.3"
+    generators = "make"
+    exports_sources = "patches/**"
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+
+        if self.settings.os == "Windows":
+            raise ConanInvalidConfiguration("Windows build not supported")
```

Comment:
I found some Visual Studio solution files in that project, I'm curious why it doesn't work.

Reply:
I did not tested it (no windows machine available) but the Vistual Studio README.txt (msvc10/cci/README.txt) list an extra dependency on boost version 1.57 (or 1.44) which is not available in conan center.